test/parallel/test-http2-binding.js

```diff
@@ -25,10 +25,9 @@ assert.strictEqual(http2.nghttp2ErrorString(-517),
 const check = Buffer.from([0x00, 0x01, 0x00, 0x00, 0x10, 0x00, 0x00, 0x04,
                            0x00, 0x00, 0xff, 0xff, 0x00, 0x05, 0x00, 0x00,
                            0x40, 0x00, 0x00, 0x02, 0x00, 0x00, 0x00, 0x01]);
-const val = http2.packSettings(http2.getDefaultSettings({}));
+const val = http2.packSettings(require('http2').getDefaultSettings());
 assert.deepStrictEqual(val, check);
 
-assert.doesNotThrow(() => assert(require('http2').getDefaultSettings()));
```

Comment:
no, those are separate operations. `packSettings()` returns a base64 encoded string with the settings, `getDefaultSettings()` returns the default settings object.